devel/summon/Portfile

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/cyberark/summon 0.9.5 v
+categories          devel
+maintainers         {@0x1DA117 danieltrautmann.com:me} openmaintainer
+license             MIT
+
+description         CLI that provides on-demand secrets access for common \
+                    DevOps tools.
+
+long_description    summon is a command-line tool to make working with secrets \
+                    easier. \
+                    It provides an interface for reading a secrets.yml file, \
+                    fetching secrets from a trusted store and exporting secret \
+                    values to a sub-process environment. \
```

Comment:
It's harmless in context, but I don't think you meant to have this final `\`.

Reply:
Oh yeah, thanks for letting me know.